### PR TITLE
Moved config.json and renamed tags

### PR DESCRIPTION
**Pull request by LucasPMagno**
Moved config.json to /data/configs/ and renamed tags to db.

### CODE REVIEW

**Comment by abyssvi** [comment]
I'd like to get home, patch the code in, and test thoroughly before I approve, risky business changing config and db files :)

**Comment by NITEHAWKDevs** [comment]
I will test later as well, does it meet all the requirements of lint?

File: src/managers/migrator.js [hunk 1]
```diff
@@ -16,4 +18,27 @@ function migrateConfig(bot, base) {
...
+    if (fse.existsSync(newPath)) {
+        try {
+            fse.moveSync(newPath, newNewPath);
+        } catch (err) {
+            console.error('Failed to migrade config.json to configs!');
+            process.exit(1);
+        }
+    }
+}
+function migrateDB(bot, base) {
```
Comment (abyssvi) [inline src/managers/migrator.js]: nitpick: newline below } above function.

File: src/managers/migrator.js [hunk 2]
```diff
@@ -3,11 +3,13 @@ const path = require('path');
...
 exports.migrate = function (bot, base) {
     migrateConfig(bot, base);
+    migrateDB(bot, base);
 };
 
 function migrateConfig(bot, base) {
     let oldPath = path.resolve(base, 'config.json');
     let newPath = path.resolve(base, '../config.json');
+    let newNewPath = path.resolve(base, '../data/configs/config.json');
     if (fse.existsSync(oldPath)) {
```
Comment (abyssvi) [inline src/managers/migrator.js]: nitpick: newline above if, below the last let.

**Pull request merged**